src/vstart.sh

```diff
@@ -227,6 +230,13 @@ case $1 in
     --short )
 	    short=1
 	    ;;
+    --crimson )
+        ceph_osd=crimson-osd
+        ;;
+    --osd-args )
```

Comment:
do we need in some places to set extra_osd_args to be empty? otherwise, if no --osd-args input, what happens?

Reply:
it's empty. please try this in your testbed.
```sh
echo ${good_news_everyone_today_is_Saturday}
```